share/instruments/instruments.xml

```diff
@@ -7378,15 +7378,15 @@
                         <head>normal</head>
                         <line>-1</line>
                         <voice>0</voice>
-                        <name>Mute High Conga</name>
+                        <name>High Tabla</name>
```

Comment:
Here we'd need to enhance the generateTS.py to also extract those strings from instruments.xml, else it'd end up untranslated